packages/mdg:camera/photo-browser.js

```diff
@@ -266,11 +266,11 @@ MeteorCamera.getPicture = function (options, callback) {
 
   closeAndCallback = function () {
     var originalArgs = arguments;
-    UI.remove(view);
+    Blaze.remove(view);
     photo.set(null);
     callback.apply(null, originalArgs);
   };
 
-  view = UI.renderWithData(Template.camera);
-  UI.insert(view, document.body);
+  view = Blaze.renderWithData(Template.camera, null, document.body);
+  // Blaze.render(view, document.body);
```

Comment:
Why should this be commented here?

Reply:
The new version doesn't require the two renderings like UI package did. Just leaving it there for posterity. Should I remove it?

Reply:
yes, please.

Reply:
Done!